.github/workflows/slash-command-dispatch.yml

```
@@ -4,6 +4,7 @@ on:
     types: [created]
 jobs:
   slashCommandDispatch:
+    if: ${{ github.event.issue.pull_request }}
```

The brackets are not needed

```suggestion
    if: github.event.issue.pull_request
```

Alternatively, we could tighten it even more to say

```suggestion
    if: github.event.issue.pull_request && contains(github.event.comment.body, '/test')
```